plog/plog

```diff
@@ -412,6 +416,16 @@ main () {
     PUSH_PROC=
     COMPRESS=
     QUIET=
+
+    set +u
+    if [[ -z "$NETRC_DIR" ]]; then
```

Comment:
You can use

    if [[ -z "${NETRC_DIR:-}" ]]; then

to safely check if a variable is set in "set -u" mode.

Reply:
Also this block will be better placed at the top with the other global variables. That way when the script is `source`d, the variables are set.